src/DIRAC/ConfigurationSystem/Client/ConfigurationClient.py

```diff
@@ -29,7 +29,7 @@ def getCompressedData(self):
         :rtype: str
         """
         retVal = self.executeRPC("getCompressedData")
-        if retVal["OK"]:
+        if retVal["OK"] and isinstance(retVal["Value"], str):
```

Comment:
shouldn't this be 
```suggestion
        if retVal["OK"] and isinstance(retVal["Value"], six.string_types):
```
?

Also below.

Reply:
Here I mean str / bytes for python2 and str for python3.
`JEncode` [encodes values](https://github.com/DIRACGrid/DIRAC/pull/5811/files#diff-292923f14bc932534035d164c48720650364237cac4df0ecabd8937a08dad029R110-R112) only if data is received in bytes, otherwise it is perceived as str and does not require encoding.

Reply:
It's probably safer to use `six.string_types`. The triggers to cause a type to be promoted to unicode can be quite fiddly.

Reply:
fixed to six.string_types